zest/releaser/utils.py

```diff
@@ -849,19 +849,32 @@ def get_last_tag(vcs, allow_missing=False):
     # by setuptools' version parsing. A direct match is obviously
     # right. The 'less' approach handles development eggs that have
     # already been switched back to development.
-    available_tags.reverse()
```

Comment:
Note: I have removed this line that reverses the available tags.  We want to go through the entire list anyway, so I don't think it matters.